src/python/grpcio/grpc/_cython/_cygrpc/call.pyx.pxi

```diff
@@ -106,6 +107,8 @@ cdef class Call:
   def __dealloc__(self):
     if self.c_call != NULL:
       grpc_call_destroy(self.c_call)
+    grpc_shutdown()
```

Comment:
```grpc_init()/grpc_shutdown()``` Should only be called at the beginning and end of the program execution.

See:
http://www.grpc.io/grpc/core/grpc_8h.html#a35f55253e80714c17f4f3a0657e06f1b